src/yb/client/client.cc

```diff
@@ -2211,7 +2211,7 @@ Result<bool> YBClient::CheckIfPitrActive() {
   return data_->CheckIfPitrActive(deadline);
 }
 
-std::vector<std::string> YBClient::ActiveUniverseHistory() {
+std::vector<WaitStateInfoPB> YBClient::ActiveUniverseHistory() {
```

Comment:
Don't have to fix this here.

But, it may be better to pass in the response/pb and have the function  "add" WaitStateInfoPBs to the passed in PB, than construct a vector of results, then the caller adds it to the pb.

Less memory allocations and copying of the auh-data.